src/test/scala/org/bitcoins/core/script/crypto/HashTypeFactoryTest.scala

```diff
@@ -8,23 +9,28 @@ import org.scalatest.{FlatSpec, MustMatchers}
 class HashTypeFactoryTest extends FlatSpec with MustMatchers  {
 
   "HashTypeFactory" must "find a hash type by its hex value" in {
-    HashTypeFactory.fromString("01") must be (SIGHASH_ALL())
-    HashTypeFactory.fromString("02") must be (SIGHASH_NONE)
-    HashTypeFactory.fromString("03") must be (SIGHASH_SINGLE)
-    HashTypeFactory.fromString("80") must be (SIGHASH_ANYONECANPAY)
+    HashTypeFactory("00000001") must be (SIGHASH_ALL(Int32.one))
+    HashTypeFactory("00000002") must be (SIGHASH_NONE)
+    HashTypeFactory("00000003") must be (SIGHASH_SINGLE)
+    HashTypeFactory("00000080") must be (SIGHASH_ANYONECANPAY)
   }
 
   it must "find a hash type by its byte value" in {
 
-    HashTypeFactory.fromByte(0x01) must be (SIGHASH_ALL())
-    HashTypeFactory.fromByte(0x02) must be (SIGHASH_NONE)
-    HashTypeFactory.fromByte(0x03) must be (SIGHASH_SINGLE)
-    HashTypeFactory.fromByte(0x80.toByte) must be (SIGHASH_ANYONECANPAY)
+    HashTypeFactory.fromBytes(Seq(0.toByte)) must be (SIGHASH_ALL(Int32.zero))
+    HashTypeFactory.fromBytes(Seq(1.toByte)) must be (SIGHASH_ALL(Int32.one))
+    HashTypeFactory.fromBytes(Seq(2.toByte)) must be (SIGHASH_NONE)
+    HashTypeFactory.fromBytes(Seq(3.toByte)) must be (SIGHASH_SINGLE)
+    HashTypeFactory.fromBytes(Int32(0x80).bytes) must be (SIGHASH_ANYONECANPAY)
   }
 
   it must "default to SIGHASH_ALL if the given string/byte is not known" in {
-    HashTypeFactory.fromByte(0x96.toByte) must be (SIGHASH_ALL(-106))
```

Comment:
This test case was incorrect. 

0x96.toByte = -106
(-106 & 0x80) = 0x80 == SIGHASH_ANYONECANPAY